EventFilter/CTPPSRawToDigi/interface/CTPPSPixelDataFormatter.h

```diff
@@ -0,0 +1,87 @@
+#ifndef CTPPSPixelDataFormatter_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h